edx_proctoring/tests/test_api.py

```diff
@@ -950,10 +950,12 @@ def test_alias_timed_out(self):
         """
 
         exam_attempt = self._create_started_exam_attempt()
+        timeout_timestamp = datetime.now(pytz.UTC) - timedelta(minutes=1)
```

Comment:
This might just be me, but I was trying to determine the significance of the 1 minute delta.  Could you:
1. Rename `timeout_timestamp` to `random_timestamp`, and
2. Subtract 3 hours, or something seemingly more random.